ts/src/binance.ts

```diff
@@ -3075,13 +3075,26 @@ export default class binance extends Exchange {
         }
         let active = (status === 'TRADING');
         if (spot) {
+            // start: Should be deprecated after the 25th of April 2024
             const permissions = this.safeList (market, 'permissions', []);
             for (let j = 0; j < permissions.length; j++) {
                 if (permissions[j] === 'TRD_GRP_003') {
                     active = false;
                     break;
                 }
             }
+            // end: Should be deprecated after the 25th of April 2024
```

Comment:
Shouldn't we remove it now? `permissionSets` is already present in the response?

Reply:
I just tested and the property `permissionSets` does not exist yet. We still have `permissions`. I can set a reminder to check after the 25th